packages/react/src/themes/teams-dark/components/Chat/chatMessageVariables.ts

```diff
@@ -8,7 +8,7 @@ export default (siteVars: any): Partial<ChatMessageVariables> => {
     contentColor: siteVars.colors.white,
     color: siteVars.colors.white,
     timestampColorMine: siteVars.colors.grey[250],
-    contentFocusOutlineColor: siteVars.colors.brand[600],
+    linkColor: siteVars.colors.brand[600],
```

Comment:
Shouldn't this be `siteVars.colors.brand[400]`? I understand that you are just renaming it, but we may as well fix it while we are defining it.

Reply:
changed to reflect correct colors